packages/pigeon/platform_tests/alternate_language_test_plugin/example/ios/RunnerTests/AllDatatypesTest.m

```diff
@@ -29,16 +29,18 @@ - (void)testAllNull {
   XCTestExpectation *expectation = [self expectationWithDescription:@"callback"];
   [api echoAllTypes:everything
          completion:^(AllTypes *_Nonnull result, NSError *_Nullable error) {
-           XCTAssertNil(result.aBool);
-           XCTAssertNil(result.anInt);
-           XCTAssertNil(result.aDouble);
-           XCTAssertNil(result.aString);
-           XCTAssertNil(result.aByteArray);
-           XCTAssertNil(result.a4ByteArray);
-           XCTAssertNil(result.a8ByteArray);
-           XCTAssertNil(result.aFloatArray);
-           XCTAssertNil(result.aList);
-           XCTAssertNil(result.aMap);
+           XCTAssertNil(error);
+           XCTAssertNotNil(result);
+           XCTAssert([result.aBool isEqual:[NSNull null]]);
```

Comment:
Can you talk about this change?  It's the sort of thing that would be better in a different PR since I imagine you are fixing some bug, it's getting lost in this big change though.

Reply:
all nil/null handling in objc has been updated since this conversation. Should be all fixed and less confusing now.